protocol/scripts/deploys/localhost/v2/deploy-localhost-fork.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# v2-core
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/02-circuit-breaker-proxy.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/03-trv-tpi-oracle.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/04-trv.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/05-trv-dusd.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/06-trv-dtemple-token.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/07-dsr-base-strategy.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/08-temple-base-strategy.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/core/99-core-post-deployment.ts
+
+# tlc
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/tlc/01-tlc-IRM-linear-kink.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/tlc/02-tlc.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/tlc/03-tlc-circuit-breaker-dai.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/tlc/04-tlc-circuit-breaker-temple.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/tlc/05-tlc-strategy.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/tlc/99-tlc-post-deployment.ts
+
+# gnosis
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/gnosis-strategy-1/01-gnosis-safe-circuit-breakers-dai.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/gnosis-strategy-1/02-gnosis-safe-circuit-breakers-temple.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/gnosis-strategy-1/03-gnosis-strategy.ts
+npx hardhat run --network localhost scripts/deploys/mainnet/v2/gnosis-strategy-1/99-post-deploy.ts
+
+# ramos
+# npx hardhat run --network localhost scripts/deploys/mainnet/v2/ramos/01-temple-dai-balancer-pool.ts
```

Comment:
I think I need to do some extra work, I'm getting some issues with the ramos local remove liquidity test, and I'm thinking deploying this should sorted out

Reply:
You don't need it :-) 
Should be able to use the existing mainnet balancer and aura pools/ids